app/assets/javascripts/pdfjs_viewer/viewer.js

```diff
@@ -6444,19 +6444,39 @@ var pdfjsWebLibs;
      });
     }
    };
+
+   // For DOBT's usage we don't want anyone to be able to load the pdf
+   // viewer as a standalone page.
+   var ensureLoadedInIframe;
+   ensureLoadedInIframe = function ensureViewerFocused() {
+     if (window.self === window.top) {
+       throw new Error('Unauthorized usage.');
+     }
+   };
    var validateFileURL;
    var HOSTED_VIEWER_ORIGINS = [
     'null',
-    'http://mozilla.github.io',
-    'https://mozilla.github.io'
+    'https://attachments.dobt.dev',
+    'https://attachments.staging.dobt.co',
+    'https://attachments.dobt.co',
+   ];
+   var FILE_ORIGIN_WHITELIST = [
+    'https://screendoor.dobt.dev',
+    'https://dobt-screendoor-staging.s3.amazonaws.com',
+    'https://dobt-screendoor.s3.amazonaws.com'
```

Comment:
It'd be pretty cool to just include this in production and the previous line in dev/staging, but definitely not necessary. I'm not sure how we'd even do that well if this is sitting in a separate gem, though.

Reply:
I decided to avoid dealing with injecting configuration into this file. The less we change this file, the easier it will be to pull in upstream updates from the pdf.js repo.